tests/time_tests/common/main.cpp

```diff
@@ -27,12 +27,21 @@ bool parseAndCheckCommandLine(int argc, char **argv) {
 }
 
 
+/**
+* @brief Function calls `runPipeline` with mandatory time tracking of full run
+*/
+int _runPipeline() {
```

Comment:
looks like the code have been added after some conflicts resolving and shouldn't be in the PR